src/utils/returns.ts

```diff
@@ -98,32 +96,28 @@ export function getMetricsForPositionWindow(positionT0: Position, positionT1: Po
   const t1Ownership = positionT0.liquidityTokenBalance / positionT1.liquidityTokenTotalSupply
 
   // get starting amounts of token0 and token1 deposited by LP
-  const token0_amount_t0 = t0Ownership * positionT0.reserve0
-  const token1_amount_t0 = t0Ownership * positionT0.reserve1
+  const token_amount_t0 = t0Ownership * positionT0.reserve
 
   // get current token values
-  const token0_amount_t1 = t1Ownership * positionT1.reserve0
-  const token1_amount_t1 = t1Ownership * positionT1.reserve1
+  const token_amount_t1 = t1Ownership * positionT1.reserve
 
   // calculate squares to find imp loss and fee differences
-  const sqrK_t0 = Math.sqrt(token0_amount_t0 * token1_amount_t0)
+  const sqrK_t0 = Math.sqrt(token_amount_t0)
   // eslint-disable-next-line eqeqeq
-  const priceRatioT1 = positionT1.token0PriceUSD != 0 ? positionT1.token1PriceUSD / positionT1.token0PriceUSD : 0
+  const priceRatioT1 = positionT1.tokenPriceUSD != 0 ? positionT1.tokenPriceUSD / positionT1.tokenPriceUSD : 0
 
-  const token0_amount_no_fees = positionT1.token1PriceUSD && priceRatioT1 ? sqrK_t0 * Math.sqrt(priceRatioT1) : 0
-  const token1_amount_no_fees =
-    Number(positionT1.token1PriceUSD) && priceRatioT1 ? sqrK_t0 / Math.sqrt(priceRatioT1) : 0
-  const no_fees_usd =
-    token0_amount_no_fees * positionT1.token0PriceUSD + token1_amount_no_fees * positionT1.token1PriceUSD
+  const token0_amount_no_fees = positionT1.tokenPriceUSD && priceRatioT1 ? sqrK_t0 * Math.sqrt(priceRatioT1) : 0
+
+  const no_fees_usd = token0_amount_no_fees * positionT1.tokenPriceUSD
+
+  const difference_fees_token0 = token_amount_t1 - token0_amount_no_fees
 
-  const difference_fees_token0 = token0_amount_t1 - token0_amount_no_fees
-  const difference_fees_token1 = token1_amount_t1 - token1_amount_no_fees
   const difference_fees_usd =
-    difference_fees_token0 * positionT1.token0PriceUSD + difference_fees_token1 * positionT1.token1PriceUSD
+    difference_fees_token0 * positionT1.tokenPriceUSD
 
   // calculate USD value at t0 and t1 using initial token deposit amounts for asset return
-  const assetValueT0 = token0_amount_t0 * positionT0.token0PriceUSD + token1_amount_t0 * positionT0.token1PriceUSD
-  const assetValueT1 = token0_amount_t0 * positionT1.token0PriceUSD + token1_amount_t0 * positionT1.token1PriceUSD
+  const assetValueT0 = token_amount_t0 * positionT0.tokenPriceUSD
+  const assetValueT1 = token_amount_t0 * positionT1.tokenPriceUSD
```

Comment:
whole logic here must be re-considered

Reply:
Since we have only one token per pool. The return can be simply calculated by (t1_amount - t0_amount) / t0_amount.

Reply:
Done.